mojaloop-vars.yaml

```diff
@@ -1,4 +1,4 @@
 mcm_chart_version: 1.2.5
 mojaloop_chart_version: 16.0.4-20851-47a121d
 finance_portal_chart_version: 5.0.0
-reporting_templates_chart_version: 1.1.12
+reporting_templates_chart_version: 1.1.14
```

Comment:
Manual version changes here (and in other files in the root folder) should not be done. In this specific case the helm version was not published and resulted in issues in ArgoCD, which was trying to deploy non-existing version. 

Instead, the patterns should be edited, which will trigger the pattern matching against existing versions and create a PR if a new version is found.